test/test_package_loading.py

```diff
@@ -34,3 +35,30 @@ def test_invalid_package_name(selenium):
     with pytest.raises(WebDriverException,
                        match="Invalid package name or URI"):
         selenium.load_package('tcp://some_url')
+
+
+def test_load_packages_multiple(selenium_standalone):
+    selenium = selenium_standalone
+    selenium.load_package(['pyparsing', 'matplotlib'])
```

Comment:
Should we test something that aren't dependencies of one another, like `matplotlib` and `py`?

Reply:
That was intentional to check that in the log output we only see `Loading pyparsing` once (it wasn't the case before changes in this PR). Pushed another commit to also test non dependent packages as you suggest.